doc/index.qdoc

```diff
@@ -95,8 +95,27 @@ CONFIG += link_pkgconfig
 PKGCONFIG += amberwebauthorization
 \endcode
 
-It may then \c{#include <oauth1.h>}, \c{#include <oauth2.h>},
-\c{#include <redirectlistener.h>} as required.
+If you're using a yaml file to package your RPM you'll also need to add
+\c{amberwebauthorization} to the \c{PkgConfigBR} section, like this:
+
+\code
+PkgConfigBR:
+  - amberwebauthorization
+\endcode
+
+Alternatively if you're writing your spec file by hand, add the following to it:
+
+\code
+BuildRequires:  pkgconfig(amberwebauthorization)
+\endcode
+
+You can then use the relevant includes as required:
+
+\code
+#include <oauth1.h>
+#include <oauth2.h>
```

Comment:
Don't these headers depend on which kind of Oauth is used version 1 or 2?
Except that LGTM.

Reply:
You're right of course. The text preceding the code snippet was supposed to imply this, but as it wasn't clear, I'll try to improve it.

Reply:
I've added some additional text to make this clearer.